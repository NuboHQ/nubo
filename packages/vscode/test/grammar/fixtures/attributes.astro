<!-- Test special characters inside attributes names -->

<div @class></div>
<div class_name></div>
<div class.astro></div>
<div i18nReady></div>
