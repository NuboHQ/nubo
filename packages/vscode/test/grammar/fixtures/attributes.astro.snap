><!-- Test special characters inside attributes names -->
#^^^^ source.astro comment.block.html punctuation.definition.comment.html punctuation.definition.comment.begin.html
#    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.astro comment.block.html
#                                                     ^^^ source.astro comment.block.html punctuation.definition.comment.html punctuation.definition.comment.end.html
>
><div @class></div>
#^ source.astro meta.tag.any.div.start.html punctuation.definition.tag.begin.html
# ^^^ source.astro meta.tag.any.div.start.html entity.name.tag.html
#    ^ source.astro meta.tag.any.div.start.html
#     ^^^^^^ source.astro meta.tag.any.div.start.html meta.attribute.@class.html entity.other.attribute-name.html
#           ^ source.astro meta.tag.any.div.start.html punctuation.definition.tag.end.html
#            ^^ source.astro meta.tag.any.div.end.html punctuation.definition.tag.begin.html
#              ^^^ source.astro meta.tag.any.div.end.html entity.name.tag.html
#                 ^ source.astro meta.tag.any.div.end.html punctuation.definition.tag.end.html
><div class_name></div>
#^ source.astro meta.tag.any.div.start.html punctuation.definition.tag.begin.html
# ^^^ source.astro meta.tag.any.div.start.html entity.name.tag.html
#    ^ source.astro meta.tag.any.div.start.html
#     ^^^^^^^^^^ source.astro meta.tag.any.div.start.html meta.attribute.class_name.html entity.other.attribute-name.html
#               ^ source.astro meta.tag.any.div.start.html punctuation.definition.tag.end.html
#                ^^ source.astro meta.tag.any.div.end.html punctuation.definition.tag.begin.html
#                  ^^^ source.astro meta.tag.any.div.end.html entity.name.tag.html
#                     ^ source.astro meta.tag.any.div.end.html punctuation.definition.tag.end.html
><div class.astro></div>
#^ source.astro meta.tag.any.div.start.html punctuation.definition.tag.begin.html
# ^^^ source.astro meta.tag.any.div.start.html entity.name.tag.html
#    ^ source.astro meta.tag.any.div.start.html
#     ^^^^^^^^^^^ source.astro meta.tag.any.div.start.html meta.attribute.class.astro.html entity.other.attribute-name.html
#                ^ source.astro meta.tag.any.div.start.html punctuation.definition.tag.end.html
#                 ^^ source.astro meta.tag.any.div.end.html punctuation.definition.tag.begin.html
#                   ^^^ source.astro meta.tag.any.div.end.html entity.name.tag.html
#                      ^ source.astro meta.tag.any.div.end.html punctuation.definition.tag.end.html
><div i18nReady></div>
#^ source.astro meta.tag.any.div.start.html punctuation.definition.tag.begin.html
# ^^^ source.astro meta.tag.any.div.start.html entity.name.tag.html
#    ^ source.astro meta.tag.any.div.start.html
#     ^^^^^^^^^ source.astro meta.tag.any.div.start.html meta.attribute.i18nReady.html entity.other.attribute-name.html
#              ^ source.astro meta.tag.any.div.start.html punctuation.definition.tag.end.html
#               ^^ source.astro meta.tag.any.div.end.html punctuation.definition.tag.begin.html
#                 ^^^ source.astro meta.tag.any.div.end.html entity.name.tag.html
#                    ^ source.astro meta.tag.any.div.end.html punctuation.definition.tag.end.html
>